packages/itwin/grouping-mapping-widget/api/grouping-mapping-widget.api.md

```diff
@@ -0,0 +1,522 @@
+## API Report File for "@itwin/grouping-mapping-widget"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="react" />
+
+import type { AccessToken } from '@itwin/core-bentley';
+import { Alert } from '@itwin/itwinui-react';
+import type { CalculatedProperty } from '@itwin/insights-client';
+import { Context } from 'react';
+import type { CustomCalculation } from '@itwin/insights-client';
+import { Dispatch } from 'react';
+import { Extraction } from '@itwin/insights-client';
+import { ExtractionStatus } from '@itwin/insights-client';
+import type { Group } from '@itwin/insights-client';
+import type { GroupProperty } from '@itwin/insights-client';
+import type { IExtractionClient } from '@itwin/insights-client';
+import type { IMappingsClient } from '@itwin/insights-client';
+import type { IModelConnection } from '@itwin/core-frontend';
+import type { Mapping } from '@itwin/insights-client';
+import { QueryClient } from '@tanstack/react-query';
+import * as React_2 from 'react';
+import { default as React_3 } from 'react';
+import type { ReactElement } from 'react';
+import { SetStateAction } from 'react';
+import { StagePanelLocation } from '@itwin/appui-react';
+import { StagePanelSection } from '@itwin/appui-react';
+import type { UiItemsProvider } from '@itwin/appui-react';
+import { UseMutateAsyncFunction } from '@tanstack/react-query';
+import type { Widget } from '@itwin/appui-react';
+
+// @public
+export const CalculatedPropertyAction: ({ mappingId, group, calculatedProperty, onSaveSuccess, onClickCancel, }: CalculatedPropertyActionProps) => JSX.Element;
+
+// @public
+export interface CalculatedPropertyActionProps {
+    // (undocumented)
+    calculatedProperty?: CalculatedProperty;
+    // (undocumented)
+    group: Group;
+    // (undocumented)
+    mappingId: string;
+    // (undocumented)
+    onClickCancel?: () => void;
+    // (undocumented)
+    onSaveSuccess: () => void;
+}
+
+// @public
+export const CalculatedPropertyActionWithVisuals: ({ mappingId, group, calculatedProperty, onSaveSuccess, onClickCancel, }: CalculatedPropertyActionWithVisualsProps) => JSX.Element;
+
+// @public
+export interface CalculatedPropertyActionWithVisualsProps {
```

Comment:
Now the question is why a separate interface is needed for the `WithVisuals` component. If it can be replaced by the `CalculatedPropertyActionProps`, then it should be done in the next PR.

Reply:
I didn't want to have them tightly coupled. In case future changes are made.